src/app/plugins/register.plugin.ts

```diff
@@ -74,26 +75,22 @@ export class RegisterPlugin extends Plugin {
     const embededMessage = new MessageEmbed();
 
     messageForUser +=
-      `\nUnable to locate the following classes: ${invalidClasses.join(' ')}` +
+      `\n${message.author}, Unable to locate the following classes: ${invalidClasses.join(' ')}\n` +
       `Below you can find suggestions for each incorrect input:`;
 
     embededMessage.setColor('#0099ff').setTitle('Atleast One Class Not Found');
     embededMessage.setDescription(messageForUser);
 
-    // I put up to nine so it's there if needed.
-    const emojis: string[] = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣'];
-
     const emojiData: IEmojiTable[] = [];
-
     invalidClasses.forEach((invalidClass: string) => {
-      const curEmote = emojis.shift() || '';
+      const curEmote = this._EMOJI_REACTIONS.shift() || '';
```

Comment:
You cant do it like that anymore, it wont work twice
i didnt like it in the first place lol
```
invalidClasses.forEach((invalidClass: string, i) => {
    const curEmote = this._EMOJI_REACTIONS[i]
    ...
});
```